Add sorting to authors dropdown on word usage tab

irina060981 opened the pull request:
For the issue https://github.com/alpheios-project/components/issues/681

balmas commented on src/vue/components/word-usage-examples/word-usage-examples-filters.vue:
```diff
@@ -85,7 +85,11 @@ export default {
+          .slice().sort((a,b) => {
+            let aT = this.calcTitle(a, 'author')
+            let bT = this.calcTitle(b, 'author')
+            return (aT < bT) ? -1 : (aT > bT) ? 1 : 0
```
we probably ought to use localeCompare here (see https://github.com/alpheios-project/components/blob/master/src/vue/components/word-list/word-language-panel.vue#L221)

irina060981 commented:
For the issue #681
1. I have updated compare method for authors and textworks on the word usage tab
2. I have added additional checks 

For the issue https://github.com/alpheios-project/components/issues/573
3. I have restyled source links to source divs
